cmux-tui/scripts/smoke-windows-long-state-path.py

```
@@ -0,0 +1,133 @@
+
+STATE_PATH_LENGTH = 423
+START_TIMEOUT_SECONDS = 20
+
+
+def make_state_path(root: Path) -> Path:
+    state = root / "state"
+    while len(str(state)) + 121 < STATE_PATH_LENGTH:
+        state /= "界" * 120
+    remaining = STATE_PATH_LENGTH - len(str(state)) - 1
+    if not 1 <= remaining <= 255:
+        raise AssertionError(f"invalid final path segment length: {remaining}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)